### ISSUE TITLE
Add `Activate Sheet` window with list of all sheets in spreadsheet

### ISSUE THREAD

**Issue by StrikerASD** [ONLYOFFICE/DocumentServer#1137]
Hello guys,

I need to ask you if is there possibility to create a feature in spreadsheet editor which expands use of sheets (the ones in the bottom of the screen). If i have low amount of spreadsheets, this isn't a problem, but if i got 20, 40 sheets and i need to scroll them many times doing my work it's painful way of work. My suggestion is (if i may :) ) that those spreadsheets can be accessible through context menu for example. If there are too many sheets there should be an icon (show all sheets for example) that by clicking it i see all spreadsheets in a window and i can scroll sheets in this window. Sounds cool and it will boost productivity. Can you reply to me if you are considering such feature?

Best regards,
Iwo Zaleski

**Comment by ShockwaveNN** [comment]
@StrikerASD Hi, is there some similar feature in MS Excel, LibreOffice or any other office software?

**Comment by StrikerASD** [comment]
No, there is such feature in google software. I will attach screen so you know how it looks like.
![sheet](https://user-images.githubusercontent.com/9042254/105956625-60b8d500-6078-11eb-909d-67d8b9e1b6ed.png)

**Comment by Rita-Bubnova** [comment]
Hello, @StrikerASD.
Thank you for enhancement.
This issue is know (issue 46767 in our private issue tracker).

**Comment by StrikerASD** [comment]
Thank you Rita :). It will be very cool if such feature will land in
onlyoffice spreadsheets.


Regards

śr., 27 sty 2021, 10:21 użytkownik Rita-Bubnova <notifications@github.com>
napisał:

**Comment by Rita-Bubnova** [comment]
Sorry, we forgot to mention that issue was fixed.
I close this issue. Feel free to comment or reopen it if you got further questions.
